osquery/tables/networking/darwin/wifi.mm

```diff
@@ -0,0 +1,163 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <Foundation/Foundation.h>
+#include <iomanip>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+
+#include "osquery/core/conversions.h"
+
+#define kAirportPreferencesPath               \
```

Comment:
Haha, I have gone back and forth on this at least a few times. I actually do prefer it as a `const std::string`, but as you mentioned converting to `{CF,NS}String` is a bit of a hassle.